packages/3dom/src/api.ts

```diff
@@ -173,6 +189,10 @@ export declare interface Material extends ThreeDOMElement {
    */
   readonly name?: string;
 
+  readonly normalTexture?: TextureInfo|null;
+  readonly occlusionTexture?: TextureInfo|null;
+  readonly emissiveTexture?: TextureInfo|null;
```

Comment:
Is there a reason the baseColorTexture and MetallicRoughnessTexture haven't been added to pbrMetallicRoughness below?

Reply:
https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#reference-material

Reply:
Sorry, I misunderstood your question. The interface for `PBRMetallicRoughness` wasn't updated to reflect the changes in this PR. I'll fix that.